language/use-cases/prod-catalog-enrichment/genai_prod_catalog_enrichment.ipynb

```diff
@@ -0,0 +1,2071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.    def get_model_response(prompt, project):
</pre><p>Why does this function exist in addition to the <code>vertex_ai_llm()</code> function? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>have made this function - <code>get_model_response</code> redundant now </p>